game/game_ui.rb

```diff
@@ -0,0 +1,52 @@
+require_relative '../game'
+require_relative '../author'
+
+class GameOptions
+  attr_accessor :games, :authors
+
+  def initialize
+    @games = []
+    @authors = []
+  end
+
+  def add_game
+    print 'Enter the publish date of the game (YYYY-MM-DD): '
+    publish_date = Date.parse(gets.chomp)
+    print 'Is the game multiplayer? [Y/N]: '
+    multiplayer_input = gets.chomp
+    multiplayer = %w[N n NO no No].include? multiplayer_input ? false : true
+    last_played_at = Date.today
+    game = Game.new(publish_date, multiplayer, last_played_at)
+    print 'Enter game author First Name: '
+    first_name = gets.chomp
+    print 'Enter game author Last Name: '
+    last_name = gets.chomp
+    author = Author.new(first_name, last_name)
+    author.add_item(game)
+    @games.push(game)
+    puts 'Game created successfully.'
+  end
+
```

Comment:
@ZewdieMc you did great.
- One optional suggestion can be to add error handling for cases where the user enters invalid input, such as an invalid date or an invalid input for the multiplayer option.